facia-tool/public/js/utils/sparklines.js

```diff
@@ -140,7 +140,10 @@ function reduceRequest (memo, front, articles, options) {
 
         return memo;
     })
-    .catch(function () {});
+    .catch(function () {
+        // Ignore errors from Ophan
+        return {};
```

Comment:
`Promise.reject(new Error('foo')).catch(function () {})` seems to work for me (using Chrome’s `Promise`). Could it be the library you're using?